azure/interfaces.go

```diff
@@ -79,6 +79,7 @@ type ClusterDescriber interface {
 	AvailabilitySetEnabled() bool
 	CloudProviderConfigOverrides() *infrav1.CloudProviderConfigOverrides
 	FailureDomains() []string
+	OutboundLBName(role string) string
```

Comment:
this is a better fit for the `NetworkDescriber` interface above (both are included in `ClusterScoper`)

Reply:
It already existed. So removed that. Thanks.